pkgs/development/python-modules/acstore/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  pyyaml,
+  setuptools,
+  ...
```

Comment:
```suggestion
```
All arguments of the package should be explicit.

Reply:
Will resolve across all packages 👍